src/router/handlers/exportHandler.ts

```diff
@@ -34,13 +35,19 @@ export default class ExportHandler {
     async cancelExport(jobId: string, requesterUserId: string): Promise<void> {
         const jobDetails = await this.bulkDataAccess.getExportStatus(jobId);
         await this.checkIfRequesterHasAccessToJob(jobDetails, requesterUserId);
+        if (['completed', 'failed'].includes(jobDetails.jobStatus)) {
```

Comment:
I think the only time you can cancel the request is when it is in 'pending' right?

Reply:
The different type of `jobStatus` are

` 'completed' | 'failed' | 'in-progress' | 'canceled' | 'canceling'`

If the job is currently in `'in-progress' | 'canceled' | 'canceling'`, we don't want to throw an error for the cancel request. We can just proceed as is.

Reply:
What will the service do when the job in canceled gets asked to cancel again?